lib/puppet/parser/functions/sqlserver_is_domain_or_local_user.rb

```diff
@@ -1,7 +1,7 @@
 require File.expand_path(File.join(File.dirname(__FILE__), '..', '..', '..', '..', 'lib/puppet_x/sqlserver/server_helper'))
 
 module Puppet::Parser::Functions
-  newfunction(:sqlserver_is_domain_or_local_user, :type => :rvalue) do |args|
+  newfunction(:sqlserver_is_domain_or_local_user, :type => :rvalue, :doc => '@return [String] Returns a string.') do |args|
```

Comment:
This function returns `true` if the username is for a local/domain user and false otherwise.